### ISSUE TITLE
Log information when caller closed the connection

### ISSUE THREAD

**Issue by querdenker2k** [quarkusio/quarkus#40879]
### Description

Currently there is no information on the server side when the caller closed the connection until the request is processed on the server.
I found no documentation how to do this. Tested this with Rest and Soap (cxf)

### Implementation ideas

Print information when the synchronous call could not be send back to sender (maybe because of a closed connection).

**Comment by geoand** [comment]
Can you please add the REST extensions you are using?

Also, how do you plan to use this information?

**Comment by querdenker2k** [comment]
This is the quarkus rest dependency: quarkus-resteasy-jackson
Currently we have a problem that the caller maybe has a very small request-timeout. The logs of the caller does not have enough information, but the server (our component) does not print any information when the client run into a timeout (tested manually with curl)

**Comment by querdenker2k** [comment]
I agree for a public deployed server scenario, but we are in closed environment and as well for 'testing' it would be nice to have the option for enabling such logging.

**Comment by geoand** [comment]
Right, that makes sense
